charmcraft/commands/pack.py

```diff
@@ -58,27 +65,93 @@ def get_paths_to_include(config):
 
 
 _overview = """
-Build the bundle and package it as a zip archive.
+Build and pack a charm operator package or a bundle.
 
-You can `juju deploy` the bundle .zip file or upload it to
-the store (see the "upload" command).
+You can `juju deploy` the resulting `.charm` or bundle's `.zip`
+file directly, or upload it to Charmhub with `charmcraft upload`.
+
+For the charm you must be inside a charm directory with a valid
+`metadata.yaml`, `requirements.txt` including the `ops` package
+for the Python operator framework, and an operator entrypoint,
+usually `src/charm.py`.  See `charmcraft init` to create a
+template charm directory structure.
+
+For the bundle you must already have the `bundle.yaml` (can be
+generated by Juju) and the README.md file.
```

Comment:
if not, charms are generic and bundles are specific :-)

```suggestion
For the bundle you must already have a `bundle.yaml` (can be
generated by Juju) and a README.md file.
```

Reply:
Thanks.,